mubench.pipeline/data/detector_run.py

```diff
@@ -66,6 +66,7 @@ def get_run_info(self):
         run_info = {
             "number_of_findings": self.number_of_findings,
             "runtime": self.runtime,
+            "timestamp": self.__timestamp
```

Comment:
Do we really need to do this? I thought that we store the run timestamp in the run info already, to trigger detector rerun if the existing run is older than the current compile? Or don't we do this?